target/linux/ath79/mikrotik/base-files/etc/board.d/01_leds

```diff
@@ -23,6 +24,9 @@ mikrotik,routerboard-sxt-5nd-r2)
 	ucidef_set_led_rssi "rssimediumhigh" "rssimediumhigh" "green:rssimediumhigh" "wlan0" "60" "100"
 	ucidef_set_led_rssi "rssihigh" "rssihigh" "green:rssihigh" "wlan0" "80" "100"
 	;;
+mikrotik,routerboard-lhg-2nd)
```

Comment:
Unrelated change?

Reply:
Weird. Maybe I screwed it during a rebase, the board is already configured in line ~9. Removing it.